Source/capture.cpp

```diff
@@ -118,7 +118,7 @@ static bool CapturePix(CelOutputBuffer buf, std::ofstream *out)
 	int height = buf.h();
 	BYTE *pBuffer = (BYTE *)DiabloAllocPtr(2 * width);
 	BYTE *pixels = buf.begin();
-	while (height--) {
+	while ((height--) != 0) {

```

Comment:
Tiny nitpick - I usually prefer > and < to != for cases like this.
Don't think clang tidy can make the connection between "decreasing loop variable" and "check exit condition with > instead of !=".
There are many such loops in the code, I'd use > for all of them.

Reply:
Turns out i'm not a fan of while: `for (int height = buf.h(); height > 0; height--) {`

Reply:
While (no pun intended) not strictly out of scope, I will probably miss most of these. But feel free to apply the change afterward. Or point it out using GitHub suggestions so I only have to click "accept".